src/util.ts

```diff
@@ -189,3 +188,5 @@ export const interpolateTimespan = (forHumans: string) => {
     });
   }
 };
+
+export const KeyProviderType = "org.keycloak.keys.KeyProvider";
```

Comment:
I think it's preferred to have our constants as screaming snake case (no joke this is what is called apparently). Otherwise it might be confused with a class or enum.
```suggestion
export const KEY_PROVIDER_TYPE = "org.keycloak.keys.KeyProvider";
```

Reply:
Great suggestion - also TIL 🐍